packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -2,18 +2,58 @@
 const expect = require('expect');
 const { accessibility } = require('../commands/validates');
 const { toBeAccessible } = require('../commands/expect');
+const viewportHelpers = require('../commands/viewport-helpers');
+const hideInputCaret = require('../commands/hide-input-caret');
 
 class TerraService {
   /**
    * Service hook executed prior to test execution.
    * Initializes the Terra Service's custom commands.
    */
-  before() {
+  before(capabilities) {
+    /* Add the Jest expect module the use the Jest matchers. */
     global.expect = expect;
     global.expect.extend({ toBeAccessible });
 
-    global.Terra = {};
-    global.Terra.validates = { accessibility };
+    /* Add a Terra global with access to Mocha-Chai test helpers. */
+    global.Terra = {
+      validates: { accessibility },
+
+      /* `viewports` provides access Terra's list of test viewports. */
+      viewports: viewportHelpers.getViewports,
+
+      /* `describeViewports` provides a custom Mocha `describe` block for looping test viewports. */
+      describeViewports: viewportHelpers.describeViewports,
+
+      /* `hideInputCaret` hides the blinking input caret that appears in inputs or editable text areas. */
+      hideInputCaret,
+    };
+
+    /* IE driver takes a longer to be ready for browser interactions. */
```

Comment:
Looks like there is an extra `a` or missing content in this comment

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/438/commits/19c190061a6d0f428d0ab4b33b669bed04d9f219